### PR TITLE
Feature/update process checklist

### PR DESCRIPTION
**Pull request by imda-peckyoke**
## Description

New Process checks given mapping with 42001

## Motivation and Context

[Explain the motivation or the context behind this pull request. Why is it necessary?]

## Type of Change

feat: update process checklist

## How to Test

1. Install updated process checklist aiverify.stock.process_checklist-0.9.3.zip.
2. Create new project with stock template
3. Verify that the changes has been made to the process checklist summaries and user inputs.

## Checklist

Please check all the boxes that apply to this pull request using "x":

- [x]  I have tested the changes locally and verified that they work as expected.
- [ ]  I have added or updated the necessary documentation (README, API docs, etc.).
- [ ]  I have added appropriate unit tests or functional tests for the changes made.
- [ ]  I have followed the project's coding conventions and style guidelines.
- [ ]  I have rebased my branch onto the latest commit of the main branch.
- [ ]  I have squashed or reorganized my commits into logical units.
- [ ]  I have added any necessary dependencies or packages to the project's build configuration.
- [x]  I have performed a self-review of my own code.
- [x]  I have read, understood and agree to the Developer Certificate of Origin below, which this project utilises.

## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]

<details>
  <summary>Developer Certificate of Origin</summary>

 ```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
 ```
</details>

### CODE REVIEW

**Comment by kimeetok** [comment]
Pending updates:
- Fix line spacing for 12.7.1
- Correct wording for 9.6.2

This should be released with PR #291

File: stock-plugins/aiverify.stock.process-checklist/inputs/config_accountability.ts [hunk 1]
```diff
@@ -78,11 +78,44 @@ export const config = {
...
+            }
+          ]
+        },
+        {
+          "testableCriteria": "Establish processes to ensure that its usage of services, products or materials are in support of the responsible development and use of the AI system",
+          "processes": [
+            {
+              "pid": "9.7.1",
+              "process": "Select AI system suppliers which align with organisation’s approach.<br/><br/>When AI system suppliers do not perform as intended, there are processes for suppliers to take remedial actions.",
+              "metric": "Internal documentation<br/>External correspondence",
```
Comment (imda-benedictlee) [inline stock-plugins/aiverify.stock.process-checklist/inputs/config_accountability.ts:99]: Can i check if metric is only "External Correspondence"

**Review by imda-benedictlee — DISMISSED** [review]
Approve with comments to trigger Integration Test

**Review by imda-benedictlee — DISMISSED** [review]
Approved

**Review by imda-benedictlee — APPROVED** [review]
Approved

**Review by HanFai-IMDA — APPROVED** [review]
Approved

**Comment by imda-benedictlee** [comment]
Pipeline Testing Passed. Screenshot Attached.
[Pipeline Testing For Dev Script.zip](https://github.com/user-attachments/files/15986261/Pipeline.Testing.For.Dev.Script.zip)
[Pipeline Testing For Docker.zip](https://github.com/user-attachments/files/15986265/Pipeline.Testing.For.Docker.zip)

**Pull request merged**